packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md

```diff
@@ -0,0 +1,155 @@
+
+> **Note:** Self-service registration and Registration Inline Hooks are only supported with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 2.9 or later.
+
+To enable the Registration Inline Hook on the self-service registration page:
+
+1. In the Admin Console, go to **Directory** > **Self-Service Registration**.
+
+1. Click **Edit**.
+
+1. From the **Extension** field drop-down menu, select the hook you've set up and activated previously ("Guide Registration Hook Code").
+
+1. Click **Save**.
+
+The Registration Inline Hook is now enabled for self-service registration. You are now ready to preview and test the example.
+
+## Preview and test
+
+The external service example is now ready with code to receive and respond to an Okta call. The Okta org is now set up to call the external service using a Registration Inline Hook.
+
+In your Okta org, you can preview the request and response JSON right from the Admin Console. You can also test the code directly with self-registering users.
+
+### Preview
+
+To preview the Registration Inline Hook:
+
+1. In the Admin Console, go to **Workflow** > **Inline Hooks**.
+2. Select the Registration Inline Hook name (in this example, "Guide Registration Hook Code").
+3. Click the **Preview** tab.
+4. Select a user from your org in the first block titled "Configure Inline Hook request"; that is, a value for the  `data.userProfile` value.
+5. From the "Preview example Inline Hook request" block, click **Generate Request**.
+    You should see the user's request information in JSON format that is sent to the external service.
+6. From the "View service's response" block, click **View Response**.
+    You will see the response from your external service in JSON format, which either allows or denies the self-registration.
+
+### Test
+
+To run a test of your Registration Inline Hook, go to the Okta sign-in page for your Okta org, click the "Sign Up" link, and attempt to self-register.
+
+* If you use an allowable email domain, such as `john@example.com`, the user registration goes through.
+* If you use an incorrect email domain, the user registration is denied; review the error message, which displays the error summary from the external service code and is passed back to Okta.
+
+> **Note:** Review [Troubleshooting hook implementations](/docs/guides/common-hook-set-up-steps/nodejs/main/#troubleshoot-hook-implementations) for information if encountering any setup or configuration difficulties.
+
+## Next steps
+
+Review the following guides to implement other Inline or Event Hook examples:
+
+* [Event Hook](/docs/guides/event-hook-implementation/)
+* [Password Import Inline Hook](/docs/guides/password-import-inline-hook/)
+* [Token Inline Hook](/docs/guides/token-inline-hook/)
+
+## See also
+
+For further background on the Registration Inline Hook, see:
+
+* [Registration Inline Hook](/docs/reference/registration-hook/) for a complete description of this Inline Hook type.
```

Comment:
Might just make the whole thing one sentence: For further background on the Registration Inline Hook, see [Registration......]().....

```suggestion
[Registration Inline Hook](/docs/reference/registration-hook/) for a complete description of this Inline Hook type.
```

Reply:
I simplified a bit to adhere to our style: 

For a complete description of this Inline Hook type, see[Registration Inline Hook](/docs/reference/registration-hook/).